cypress/e2e/tests/navigation/not-found-page.spec.ts

```diff
@@ -1,6 +1,6 @@
 import NotFoundPagePo from '@/cypress/e2e/po/pages/not-found-page.po';
 
-describe('Not found page display', () => {
+describe('Not found page display', { tags: ['@adminUser', '@standardUser'] }, () => {
```

Comment:
Comment only. Cripes none of these were running? We should have an issue to error if there are e2e jobs that aren't run

Reply:
I've written an issue for this now: https://github.com/rancher/dashboard/issues/9350